partitura/score.py

```diff
@@ -314,13 +311,14 @@ def measure_number_map(self):
             measures = np.array([(t0, tN, 1)])
 
         inter_function = interp1d(
-            measures[:, 0], measures[:, 2], kind="previous", fill_value="extrapolate"
+            measures[:, 0],
+            measures[:, 2],
+            kind="previous",
+            fill_value="extrapolate",
+            dtype=int,
         )
 
-        def int_interp1d(input):
```

Comment:
these lines were removed since this case is covered by the new `interp1d` utility in `partitura.utils.generic`. (See the added `dtype=int` above)